website/templates/util/render_node.mako

```diff
@@ -51,16 +51,26 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
```

Comment:
I'm probably wrong but it does not seem to be related to archiveJob. It seems to be for compatibility reasons. https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L530